src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -911,7 +911,7 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         /// <summary>
         /// Cancellation token source.
         /// </summary>
-        private CancellationTokenSource _cancelToken = null;
+        internal CancellationTokenSource _cancelToken = null;
```

Comment:
The member name is not perfect :) it's not a token. But let's keep it unchanged in this PR.